UISwitch/UISwitch/ContentView.swift

```diff
@@ -0,0 +1,247 @@
+                                }
+                            }
+                        }
+                        .padding(10)
+                        Divider()
+                            .frame(width: 170)
+                        Button(action: {
+                            currentCardName = thirdCardName
+                            currentBalance = thirdCardBalance
+                        }) {
+                            HStack {
+                                Image(Constants.ImageName.cardImageName)
+                                VStack(alignment: .leading) {
+                                    Text(String(thirdCardBalance) + "0 ₽")
+                                        .font(.system(size: 14))
+                                        .fontWeight(.heavy)
+                                    Text(Constants.Text.thirdCardNumber)
+                                        .font(.system(size: 10))
+                                        .fontWeight(.medium)
+                                }
+                            }
+                        }
+                        .padding(10)
+                        Divider()
+                            .frame(width: 170)
+                        Spacer()
+                    }
+                    .foregroundColor(.black)
+                    .padding()
+                    Spacer()
+                }
+                RoundedRectangle(cornerSize: .zero)
+                    .fill(.myCardAvailable)
+                    .offset(x: isOnToggle ? 200 : 0)
+                    .overlay (
+                        VStack(alignment: .leading) {
+                            Text(currentCardName)
+                                .font(.system(size: 20))
+                                .fontWeight(.bold)
+                                .padding(.top, 30)
+                                .padding(.leading)
+                                .offset(x: isOnToggle ? 200 : 0)
+                            ZStack {
+                                RoundedRectangle(cornerRadius: 20)
+                                    .fill(Color.myGreen)
+                                    .frame(width: 360, height: 200)
+                                    .offset(x: isOnToggle ? 200 : 0)
+                                HStack {
+                                    VStack (alignment: .leading){
+                                        Text(String(currentBalance) + "0 ₽")
+                                            .font(.system(size: 20))
+                                            .fontWeight(.bold)
+                                            .foregroundStyle(.white)
+                                            .offset(x: isOnToggle ? 200 : 0)
+                                        Text(Constants.Text.availableText)
+                                            .foregroundStyle(.myCardAvailable)
+                                            .offset(x: isOnToggle ? 200 : 0)
```

Comment:
Если офсет применяется к нескольким вьюхам одного стека, то лучше применить офсет для всего стека
```suggestion
                                            .offset(x: isOnToggle ? 200 : 0)
```